nodeJS/orms/prisma_orm.md

```diff
@@ -118,6 +118,14 @@ You might have a complex query that you just are unable to get right via the Pri
 
 Prisma migrate is a tool that helps you perform database migrations. You won't be using it a whole ton in the curriculum, but it's good to be aware of it. When you decide to change the schema in any way, you run a Prisma migration to apply the schema changes to the database. These changes are tracked in a `migrations` folder in your codebase.
 
+<div class="lesson-note" markdown="1">
+
+#### ORM Limitations
```

Comment:
Nit both to be specific about Prisma ORM and not ORMs in general, and our [headings must use sentence case](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#case).
```suggestion
#### Prisma ORM limitations
```